infra/modules/worklytics-connector-specs/main.tf

```diff
@@ -260,8 +260,6 @@ locals {
       target_host : "graph.microsoft.com"
       required_oauth2_permission_scopes : []
       required_app_roles : [
-        "OnlineMeetings.Read.All",
-        "OnlineMeetingArtifact.Read.All",
```

Comment:
shouldn't this also be reflected in docs updates? docs/README, and the outlook-cal specific docs page w scopes?